vNext/extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -492,25 +482,39 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
                 // so, if they exist use only them, otherwise use request's because they should have been applied in the first place
                 // not using original request headers will result in them being lost
                 init.headers = new Headers(init.headers || (input instanceof Request ? (input.headers || {}) : {}));
-                init.headers.set(RequestHeaders.requestIdHeader, ajaxData.id);
-                if (this._config.enableRequestHeaderTracking) {
-                    ajaxData.requestHeaders[RequestHeaders.requestIdHeader] = ajaxData.id;
+                if (this._config.distributedTracingMode === DistributedTracingModes.AI || this._config.distributedTracingMode === DistributedTracingModes.AI_AND_W3C) {
```

Comment:
nit: refactor these 2 `if`'s to use a variable of `isUsingW3CHeaders` and `isUsingAIHeaders`